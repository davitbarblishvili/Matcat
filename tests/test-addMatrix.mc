func main() int 
{
    matrix a; 
    matrix b; 
    matrix c;

    a = [[1,2,3],[4,5,6]];
    b = [[3,2,1],[6,5,4]];

    c = a + b;
    printm(c);
    return 0;

}
