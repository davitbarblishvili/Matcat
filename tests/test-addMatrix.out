(
[4 4 4 ]
[5 8 11 ]
[2 4 6 ]
)